### DISCUSSION TITLE
Resume functionality when training locally via command line

### DISCUSSION THREAD

**Discussion by jacklion710** [acids-ircam/RAVE#194] [Q&A]
Can anyone explain whether or not it's possible to resume training if it is interrupted during training? Specifically on a local device as opposed to on the cloud like in colab. I started using my DAW while training and it stopped training prematurely. If so can someone explain how to do this clearly? Thanks

**Comment by potrepka** [comment]
I would also like to know this as well, as I had my power go out while I was training.

The train.py file seems to indicate to pass a checkpoint file, but https://github.com/acids-ircam/RAVE/discussions/75#discussion-4101726 says that a directory should be passed instead.
